devicetypes/smartthings/zigbee-lock.src/zigbee-lock.groovy

```diff
@@ -176,11 +186,20 @@ def poll() {
  */
 def refresh() {
 	log.trace "ZigBee DTH - Executing refresh() for device ${device.displayName}"
-	def cmds =
+	
+	if (device.getDataValue("model") == "E261-KR0B0Z0-HA") {
+		def cmds = []
+		cmds += zigbee.readAttribute(CLUSTER_IAS_ZONE, IAS_ATTR_ZONE_STATUS)
+		cmds += zigbee.readAttribute(CLUSTER_DOORLOCK, DOORLOCK_ATTR_LOCKSTATE)
+		cmds += zigbee.enrollResponse()
+		return cmds
+	} else {
+		def cmds =
 		zigbee.readAttribute(CLUSTER_DOORLOCK, DOORLOCK_ATTR_LOCKSTATE) +
 		zigbee.readAttribute(CLUSTER_POWER, POWER_ATTR_BATTERY_PERCENTAGE_REMAINING)
-	log.info "ZigBee DTH - refresh() returning with cmds:- $cmds"
-	return cmds
+		log.info "ZigBee DTH - refresh() returning with cmds:- $cmds"
```

Comment:
Move the log and return out of the `else` so it handles the `"E261-KR0B0Z0-HA"` case too

Reply:
ok